yb-voyager/src/namereg/namereg.go

```diff
@@ -34,6 +32,28 @@ const (
 
 var NameReg NameRegistry
 
+type SourceDbRegistry interface {
+	GetAllTableNamesRaw(schemaName string) ([]string, error)
+}
+
+type YBDBRegistry interface {
```

Comment:
We should not use word "Registry" in these interface names. Let's call them just `SourceDBInterface` and `YBDBInterface`.